tracer/src/Datadog.Tracer.Native/iast/dataflow.cpp

```diff
@@ -506,7 +511,13 @@ ModuleInfo* Dataflow::GetModuleInfo(ModuleID id)
     {
         return found->second;
     }
-    return nullptr;
+
+    ModuleInfo* pModuleInfo = nullptr;
+    if (SUCCEEDED(ModuleLoaded(id, &pModuleInfo)))
+    {
+        _modules[id] = pModuleInfo;
+    }
+    return pModuleInfo;
```

Comment:
is `pModuleInfo` guaranteed to be `nullptr` if `ModuleLoaded` did _not_ succeed? if not, should we return `nullptr` directly to be sure?
```suggestion
    if (SUCCEEDED(ModuleLoaded(id, &pModuleInfo)))
    {
        _modules[id] = pModuleInfo;
        return pModuleInfo;
    }
    return nullptr;
```

Reply:
In fact, the map assignation was not needed. Fixed in f5493a979e2bfb6324dc0286422a8400d7543d3b